File: WebAPI/GerenciadorProjetos/Models/Planejamento/Tarefa.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace GerenciadorProjetos.Models.Planejamento
{
    [Table("Tarefa")]
    public class Tarefa
    {
        public int TarefaID { get; set; }

        [Required]
        public string Descricao { get; set; }

        public double? Ordem { get; set; }

        public int AtividadeID { get; set; }
        public Atividade Atividade { get; set; }
    }
}

File: VueApp/gerenciador-projetos/src/domain/tarefa/Tarefa.js
export default class Tarefa {

    constructor(tarefaID = 0, descricao = '', ordem = '', atividadeID = 0) {

        this.tarefaID = tarefaID;
        this.descricao = descricao;
        this.ordem = ordem;
        this.atividadeID = atividadeID;
    }

    mapear(tarefa = { tarefaID: 0, descricao: '', ordem: '', atividadeID: 0 }) {

        this.tarefaID = tarefa.tarefaID;
        this.descricao = tarefa.descricao;
        this.ordem = tarefa.ordem;
        this.atividadeID = tarefa.atividadeID;
    }
}